### DISCUSSION TITLE
5V USB-powered version?

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#64] [Ideas]
Just putting this idea out there to gather feedback.

Many folks plan to use MAIANA with a laptop running OpenCPN or something similar, and the main interface they want is a USB/serial COM port. The breakout board that will ship with the first kits will have this.

Would there be a benefit if the unit dropped the 12V requirement altogether? To be transparent, the PA in the circuit cannot run from 5V, and the amount of power needed during a 26 millisecond transmission is more than a typical USB port can handle. Also, the cable interface between the antenna and the cabin should not be USB -- it will be far too long for that.

But all of these issues can be overcome with a 5V->12V boost and a supercapacitor in the unit (which I actually tested years ago).  Do you think it makes sense to create this variant?

**Comment by javelin911** [comment]
It is an interesting thought and would simplify wiring for people that want
to use this with say a Raspberry Pi with openCPN.
I personally don't care if it has 12v and usb.  I plan to put it in one
place and leave it there, so the extra wire is not a huge deal.

On Tue, Jun 8, 2021 at 1:54 PM Peter Antypas ***@***.***>
wrote:

> Just putting this idea out there to gather feedback.
>
> Many folks plan to use MAIANA with a laptop running OpenCPN or something
> similar, and the main interface they want is a USB/serial COM port. The
> breakout board that will ship with the first kits will have this.
>
> Would there be a benefit if the unit dropped the 12V requirement
> altogether? To be transparent, the PA in the circuit cannot run from 5V,
> and the amount of power needed during a 26 millisecond transmission is more
> than a typical USB port can handle. Also, the cable interface between the
> antenna and the cabin should not be USB -- it will be far too long for that.
>
> But all of these issues can be overcome with a 5V->12V boost and a
> supercapacitor in the unit (which I actually tested years ago). Do you
> think it makes sense to create this variant?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/peterantypas/maiana/discussions/64>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACVWXJS2FUJ4W7FRL4OQEI3TRZKLTANCNFSM46KNSC5Q>
> .
>


-- 
Robert Baxter
***@***.***
571-723-1502 - cell

**Comment by jrcardwell** [comment]
What about using POE to provide power and signals that can then connect to a router and broadcast the signals via wifi?  I use POE for connecting a wifi booster to the cabin in order to share the wifi with a base station.  I also use a raspberry Pi multiplexer that connects via nmea0183-to-USB to my current AIS receiver and sends that via Ethernet to the router so that my iPad can receive the AIS signals in iNavX over wifi.

**Comment by Rudder2** [comment]
I like the idea of it running on 5 volts because when my batteries got low, because of failure and no way to charge or fix to i got to shore, my navigation system stayed up because it ran on 5 volts and my Buck converter converts 9 - 36 volts to 5v. 

On the flip side i would of probably turned off AIS to save power for more important systems.

**Comment by marcelrv** [comment]
Personally I think it may be useful, as long as it does not drop the possibility to power it from 12V, which is ready available on board on virtually every boat...5V not.

Having a buck/boost converter that allows to be very flexible with the input voltage indeed (possibly even serving 24V which is on some boats) is an advantage

**Comment by peterantypas** [comment]
Thanks everyone. I think it's clear that this is not a path worth pursuing.

**Comment by peterantypas** [comment]
With USB back-powering and the need for galvanic isolation, I think it's fair to say that MAIANA will never be USB powered.

**Comment by jafrado** [comment]
Have you considered USB-C? I've tested the below with radios which pull 900ma under load at over 100K AGL. 
![image](https://user-images.githubusercontent.com/11479564/127578091-82065ad4-367a-449f-972b-1b4f2a29afb8.png)
Also, with pads one may be able to equip with an IP67USB-C connector to accommodate and possibly remain sea-worthy.
https://www.bulgin.com/products/pub/media/bulgin/data/4000_series_c_type.pdf

The PXP4043/C I would probably snip the cables and solder both D+/D- to the pads. Then, I could mount the connector to a panel.

Thank you very much for all of your great work, I definitely want one when they become available :+1:

**Reply by peterantypas** [reply]
The main issue with USB is not the current but the voltage. 5V will not run the PA. It needs a (theoretical) minimum of 7.5V, and I have it dialed at 8.1 right now. So instead of a buck, now we'll need a high frequency boost instead.

A secondary issue is that we lose isolation, but this doesn't affect everyone.

**Reply by peterantypas** [reply]
So the main question to answer, I think, is:

"What is a good use case for a USB-powered version of MAIANA"?

I don't think we have a good answer yet.

**Comment by jafrado** [comment]
Ah, I see. Well easy - ship to shore communications, software development,
and external display (e.g. HUD).

**Reply by peterantypas** [reply]
OK, you want to expand on this? Let's list some like this:

"As I user I want to be able to .... "

**Reply by jafrado** [reply]
As a user I want to be able to:

- Cluster and power devices over USB-C to build an antenna array supporting spatial-diversity. 
- Build a chart-plotter/display device which uses a USB input port for power and control.
- Host support - e.g. USB HID or CDC device driver and SW library as core hardware abstraction layer/API.
- Software application support - use a Mobile device to run control & charting software.
